app/metrics.yaml

```diff
@@ -16,10 +16,70 @@ distribution:
       - baseline
       - events
       - metrics
+      - session_end
     bugs:
       - https://github.com/MozillaReality/FirefoxReality/issues/1420
     data_reviews:
       - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
     notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+url:
+  domains:
+    type: counter
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many domains are visited in a session.
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  visits:
+    type: counter
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many URL links are visited in a session.
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  query_type:
```

Comment:
Our team would like to make `query_type` to be `lifetime:ping` and send it daily. @Dexterp37 Do you think if I need to move it out of `url` category because others are sent in `session_end`?

Reply:
No, I think it's fine to leave it in that category, no worries! Categories are meant as a way to organize semantically related data, so no worries! Things with different lifetimes can be under the same category.

That's really up to you and how do you want this file, _your file_, to be structured :) One question that might inform your choice is the following: "Would I change the name of the category or the metric if I changed the lifetime?". In general, you would want the metric identifier (category + name) to be about what the metric is, not when the metric gets cleared.

So, to summarise, my personal opinion would be to leave the metric in this category, just changing the lifetime.